### PR TITLE
Add Turkish translation

### PR DESCRIPTION
**Pull request by bitigchi**
This PR adds Turkish translations.

I would also like to update Unicode language names, but I was not sure how they were auto-generated from the Glossary file.

I plan to translate the help content in the coming weeks as well.

### CODE REVIEW

File: CotEditor/tr.lproj/PrintPane.strings [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+"939.title" = "Yazdırma Tarihi";
+
+/* Class = "NSMenuItem"; title = "Page Number"; ObjectID = "940"; */
+"940.title" = "Sayfa Numarası";
+
+/* Class = "NSMenuItem"; title = "None"; ObjectID = "941"; */
+"941.title" = "Hiçbiri";
+
+/* Class = "NSMenuItem"; title = "Don’t Print"; ObjectID = "946"; */
+"946.title" = "Yazdırma!";
```
Comment (1024jp) [inline CotEditor/tr.lproj/PrintPane.strings:63]: Sorry, I don't know about Turkish, but does `!` need here?
(Same for the same term below and in the PrintPanelAccessories.string.)

File: CotEditor/tr.lproj/DocumentInspectorView.strings [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+"gzD-jh-yod.title" = "768 (256)";
+
+/* Class = "NSTextFieldCell"; title = "Created"; ObjectID = "hgp-Cc-KHJ"; */
+"hgp-Cc-KHJ.title" = "Oluşturma";
+
+/* Class = "CocoaBindingsConnection"; ibShadowedIsNilPlaceholder = "-"; ObjectID = "igM-5I-vUh"; */
+"igM-5I-vUh.ibShadowedIsNilPlaceholder" = "-";
+
+/* Class = "NSTextFieldCell"; title = "Unicode"; ObjectID = "kna-yU-liB"; */
+"kna-yU-liB.title" = "Unicode";
```
Comment (1024jp) [inline CotEditor/tr.lproj/DocumentInspectorView.strings:120]: This field was changed from "Unicode" to "Code Points" in one of the latest commits 7a3676ada101c43e60375b106b3519844c7fdf4b. Could you suggest a new localized term for "Code Points"?

File: CotEditor/tr.lproj/HUDView.strings [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (1024jp) [inline CotEditor/tr.lproj/HUDView.strings:1]: This file can be deleted.

File: CotEditor/tr.lproj/WebDocumentWindow.strings [hunk 4]
```diff
@@ -0,0 +1,3 @@
+
+/* Class = "NSWindow"; title = "Window"; ObjectID = "MlV-Mu-4n5"; */
+"MlV-Mu-4n5.title" = "Window";
```
Comment (1024jp) [inline CotEditor/tr.lproj/WebDocumentWindow.strings:3]: This file can be deleted.

File: CotEditor.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -4051,10 +4148,11 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = "Accent Color";
 				CURRENT_PROJECT_VERSION = 474;
+				DEVELOPMENT_TEAM = D4YF34WLDG;
 				ENABLE_HARDENED_RUNTIME = YES;
 				INFOPLIST_FILE = CotEditor/Info.plist;
 				MARKETING_VERSION = "4.1.0-alpha";
-				PRODUCT_BUNDLE_IDENTIFIER = "com.coteditor.$(PRODUCT_NAME)";
+				PRODUCT_BUNDLE_IDENTIFIER = com.bitigchi.CotEditor;
```
Comment (1024jp) [inline CotEditor.xcodeproj/project.pbxproj:4155]: Please revert the change here.

File: CotEditor.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -4066,10 +4164,11 @@
...
```
Comment (1024jp) [inline CotEditor.xcodeproj/project.pbxproj:4171]: Please revert the change here.

**Review by 1024jp — CHANGES_REQUESTED** [review]
Please review my comment and update the changes if needed.

**Comment by 1024jp** [comment]
@bitigchi Thank you for the PR and the big work!
I am really excited to add the new localization to CotEditor. I appreciate.

I guess you have already done but if not yet, please check the [localization part in CONTRIBUTING.md](https://github.com/coteditor/CotEditor/blob/develop/CONTRIBUTING.md#localization).

> Continuous maintenance of the localization is highly recommended when providing a new localization. Please tell us if you can work with us. We'll call you every time before releasing a new version when we have new strings to be localized so that you can keep all your localized strings up to date.

Related to this section, I would like to ask you whether you intend to maintain the localization in the future. If yes, I'll mention you in GitHub when new strings to localize are added.

> I would also like to update Unicode language names, but I was not sure how they were auto-generated from the Glossary file.

Don't worry. I'll do it at first and then ask you if I have some questions in Turkish.

> I plan to translate the help content in the coming weeks as well.

I personally do not recommend localizing the help contents because of the maintenance cost. It is a big work that requires a lot of care and attention to the terminology. (Furthermore, actually, just a few people use the legacy Help viewer even it is useful) However, if you want to try I'll take your future PR.

Instead, it is kind, if you also make a PR for the App Store description in the [Documents-for-AppStore](https://github.com/coteditor/Documents-for-AppStore) repo.

File: CotEditor/tr.lproj/Acknowledgments.html [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+
+Her hakkı saklıdır.
+
+Aşağıdaki koşulların karşılanması koşuluyla, değişiklik yapılarak veya yapılmadan kaynak ve ikili biçimlerde yeniden dağıtıma ve kullanıma izin verilir:
+1. Kaynak kodunun yeniden dağıtımlarında yukarıdaki telif hakkı bildirimi, bu koşullar listesi ve aşağıdaki sorumluluk reddi beyanı bulunmalıdır.
+2. İkili biçimde yeniden dağıtımlar, dağıtımla birlikte sağlanan belgelerde ve/veya diğer materyallerde yukarıdaki telif hakkı bildirimini, bu koşullar listesini ve aşağıdaki sorumluluk reddini yeniden üretmelidir.
+
+BU YAZILIM YAZAR TARAFINDAN "OLDUĞU GİBİ" SAĞLANMAKTADIR VE SATILABİLİRLİK VE BELİRLİ BİR AMACA UYGUNLUK DAHİL ANCAK BUNLARLA SINIRLI OLMAYAN AÇIK VEYA ZIMNİ HERHANGİ BİR GARANTİ VERİLMEZ. YAZAR HİÇBİR DURUMDA HERHANGİ BİR DOĞRUDAN, DOLAYLI, ARIZİ, ÖZEL, ÖRNEK OLARAK YA DA DOLAYLI HASARLARDAN (YİNE DEĞİŞEN MAL VEYA HİZMET SATIN ALMASI, İŞ KAYBI, KAR KAYBI) NEDEN OLURSA OLSUN VE HERHANGİ BİR SORUMLULUK TEORİSİNE GÖRE, OLASILIK TAVSİYE EDİLMİŞ OLSA BİLE, SÖZLEŞME, KESİN SORUMLULUK VEYA HAKSIZ FİİL (İHMAL VEYA BAŞKA BİR ŞEKİLDE) BU YAZILIMIN KULLANIMINDAN DOĞAN HİÇBİR SORUMLULUĞU KABUL ETMEZ.
+</pre></blockquote>
```
[the remark this answers is not in the document]
Comment (bitigchi) [reply CotEditor/tr.lproj/Acknowledgments.html]: Reverted.

File: CotEditor/tr.lproj/Acknowledgments.html [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+<summary>MIT lisansı</summary>
+<blockquote><pre>
+© 2014-2020 1024jp
+
+İşbu belgeyle, bu yazılımın ve ilgili belge dosyalarının ("Yazılım") bir kopyasını edinen herhangi bir kişiye, herhangi bir sınırlama olmaksızın, kullanım, kopyalama, değiştirme, Yazılım'ın kopyalarını birleştirme, yayınlama, dağıtma, alt lisansını verme ve/veya satma ve aşağıdaki koşullara tabi olarak Yazılım'ın sağlandığı kişilerin bunu yapmasına izin verme:
+
+Yukarıdaki telif hakkı bildirimi ve bu izin bildirimi, Yazılım'ın tüm kopyalarına veya önemli bölümlerine dahil edilecektir.
+
+YAZILIM, SATILABİLİRLİK, BELİRLİ BİR AMACA UYGUNLUK VE İHLAL ETMEME GARANTİLERİ DAHİL ANCAK BUNLARLA SINIRLI OLMAYAN AÇIK VEYA ZIMNİ HİÇBİR GARANTİ VERİLMEKSİZİN "OLDUĞU GİBİ" SAĞLANMAKTADIR. HİÇBİR DURUMDA YAZARLAR VEYA TELİF HAKKI SAHİPLERİ, SÖZLEŞME, HAKSIZ VEYA BAŞKA BİR DURUMDA, YAZILIM'IN DIŞINDAKİ VEYA BAĞLANTILI OLARAK KULLANIMINDAN VEYA YAZILIM'LARDAKİ HERHANGİ BİR İDDİA, ZARAR VEYA BAŞKA YÜKÜMLÜLÜKLERDEN SORUMLU DEĞİLDİR.
+</pre></blockquote>
```
[the remark this answers is not in the document]
Comment (bitigchi) [reply CotEditor/tr.lproj/Acknowledgments.html]: Reverted.

File: CotEditor/tr.lproj/Acknowledgments.html [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+        <li>Hiroto Sakai &lt;<a href="http://www.fan.gr.jp/~sakai/">http://www.fan.gr.jp/~sakai/</a>&gt; </li>
+    </ul></li>
+</ul>
+</section>
+</section>
+
+
+<section id="Proje">
+<h2>CotEditor Projesi</h2>
+<p>CotEditor orijinal olarak nakamuxu &lt;<a href="https://www.aynimac.com">https://www.aynimac.com</a>&gt; tarafından yapılmıştır ve şimdi 1024jp &lt tarafından geliştirilmektedir. ;<a href="http://wolfrosch.com">http://wolfrosch.com</a>&gt;.</p>
```
[the remark this answers is not in the document]
Comment (bitigchi) [reply CotEditor/tr.lproj/Acknowledgments.html]: Fixed.

File: CotEditor/tr.lproj/HUDView.strings (hunk 3 above)
[the remark this answers is not in the document]
Comment (bitigchi) [reply CotEditor/tr.lproj/HUDView.strings]: Deleted.

File: CotEditor/tr.lproj/MultipleReplacementPanel.strings [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+"QRo-Az-lq4.ibShadowedToolTip" = "Değişim dizisinde meta karakterlerini ters eğik çizgi ile kaçırma.";
+
+/* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "RZp-5N-6CY"; */
+"RZp-5N-6CY.title" = "Text Cell";
+
+/* Class = "NSTableColumn"; headerCell.title = "Description"; ObjectID = "Rhy-iE-CIC"; */
+"Rhy-iE-CIC.headerCell.title" = "Açıklama";
+
+/* Class = "NSTableColumn"; headerCell.title = "RE"; ObjectID = "VtR-Qz-MBf"; */
+"VtR-Qz-MBf.headerCell.title" = "Dİ";
```
[the remark this answers is not in the document]
Comment (bitigchi) [reply CotEditor/tr.lproj/MultipleReplacementPanel.strings:105]: I'd leave Dİ as-is, since RE does not ring any bells for a Turkish speaker.

File: CotEditor/tr.lproj/PrintPane.strings (hunk 1 above)
Comment (bitigchi) [reply CotEditor/tr.lproj/PrintPane.strings]: It's not absolutely necessary, deleted.

File: CotEditor/tr.lproj/DocumentInspectorView.strings (hunk 2 above)
Comment (bitigchi) [reply CotEditor/tr.lproj/DocumentInspectorView.strings]: Changed.

File: CotEditor/tr.lproj/WebDocumentWindow.strings (hunk 4 above)
Comment (bitigchi) [reply CotEditor/tr.lproj/WebDocumentWindow.strings]: Deleted.

File: CotEditor.xcodeproj/project.pbxproj (hunk 5 above)
Comment (bitigchi) [reply CotEditor.xcodeproj/project.pbxproj]: Oops. Reverted.

File: CotEditor.xcodeproj/project.pbxproj (hunk 6 above)
Comment (bitigchi) [reply CotEditor.xcodeproj/project.pbxproj]: Reverted.

**Comment by bitigchi** [comment]
> I guess you have already done but if not yet, please check the [localization part in CONTRIBUTING.md](https://github.com/coteditor/CotEditor/blob/develop/CONTRIBUTING.md#localization).
> 
> > Continuous maintenance of the localization is highly recommended when providing a new localization. Please tell us if you can work with us. We'll call you every time before releasing a new version when we have new strings to be localized so that you can keep all your localized strings up to date.
> 
> Related to this section, I would like to ask you whether you intend to maintain the localization in the future. If yes, I'll mention you in GitHub when new strings to localize are added.

大丈夫、コットエディトールが大好き。いつだってアップデートをします。

> > I would also like to update Unicode language names, but I was not sure how they were auto-generated from the Glossary file.
> 
> Don't worry. I'll do it at first and then ask you if I have some questions in Turkish.

はい。

> > I plan to translate the help content in the coming weeks as well.
> 
> I personally do not recommend localizing the help contents because of the maintenance cost. It is a big work that requires a lot of care and attention to the terminology. (Furthermore, actually, just a few people use the legacy Help viewer even it is useful) However, if you want to try I'll take your future PR.

分かりました。しかし、まだ翻訳たいけど。

> Instead, it is kind, if you also make a PR for the App Store description in the [Documents-for-AppStore](https://github.com/coteditor/Documents-for-AppStore) repo.

します。翻訳物を受け入れてありがとうございます。

File: CotEditor/tr.lproj/HUDView.strings (hunk 3 above)
Comment (bitigchi) [reply CotEditor/tr.lproj/HUDView.strings]: Deleted.

File: CotEditor/tr.lproj/MultipleReplacementPanel.strings (hunk 10 above)
[the remark this answers is not in the document]
Comment (1024jp) [reply CotEditor/tr.lproj/MultipleReplacementPanel.strings:105]: Ok, thank you for the answer.

File: CotEditor/tr.lproj/Acknowledgments.html [hunk 11]
```diff
@@ -0,0 +1,214 @@
...
+<summary>MIT lisansı</summary>
+<blockquote><pre>
+Telif hakkı (c) 2016 JP Simard.
+
+İşbu belgeyle, bu yazılımın ve ilgili belge dosyalarının ("Yazılım") bir kopyasını edinen herhangi bir kişiye, herhangi bir sınırlama olmaksızın, kullanım, kopyalama, değiştirme, Yazılım'ın kopyalarını birleştirme, yayınlama, dağıtma, alt lisansını verme ve/veya satma ve aşağıdaki koşullara tabi olarak Yazılım'ın sağlandığı kişilerin bunu yapmasına izin verme:
+
+Yukarıdaki telif hakkı bildirimi ve bu izin bildirimi, Yazılım'ın tüm kopyalarına veya önemli bölümlerine dahil edilecektir.
+
+YAZILIM, SATILABİLİRLİK, BELİRLİ BİR AMACA UYGUNLUK VE İHLAL ETMEME GARANTİLERİ DAHİL ANCAK BUNLARLA SINIRLI OLMAYAN AÇIK VEYA ZIMNİ HİÇBİR GARANTİ VERİLMEKSİZİN "OLDUĞU GİBİ" SAĞLANMAKTADIR. HİÇBİR DURUMDA YAZARLAR VEYA TELİF HAKKI SAHİPLERİ, SÖZLEŞME, HAKSIZ VEYA BAŞKA BİR DURUMDA, YAZILIM'IN DIŞINDAKİ VEYA BAĞLANTILI OLARAK KULLANIMINDAN VEYA YAZILIM'LARDAKİ HERHANGİ BİR İDDİA, ZARAR VEYA BAŞKA YÜKÜMLÜLÜKLERDEN SORUMLU DEĞİLDİR.
+</pre></blockquote>
```
[the remark this answers is not in the document]
Comment (bitigchi) [reply CotEditor/tr.lproj/Acknowledgments.html]: Reverted.

File: CotEditor/tr.lproj/Acknowledgments.html [hunk 12]
```diff
@@ -0,0 +1,214 @@
...
+<summary>MIT lisansı</summary>
+<blockquote><pre>
+Telif hakkı (c) 2011 Ethan Schoonover
+
+İşbu belgeyle, bu yazılımın ve ilgili belge dosyalarının ("Yazılım") bir kopyasını edinen herhangi bir kişiye, herhangi bir sınırlama olmaksızın, kullanım, kopyalama, değiştirme, Yazılım'ın kopyalarını birleştirme, yayınlama, dağıtma, alt lisansını verme ve/veya satma ve aşağıdaki koşullara tabi olarak Yazılım'ın sağlandığı kişilerin bunu yapmasına izin verme:
+
+Yukarıdaki telif hakkı bildirimi ve bu izin bildirimi, Yazılım'ın tüm kopyalarına veya önemli bölümlerine dahil edilecektir.
+
+YAZILIM, SATILABİLİRLİK, BELİRLİ BİR AMACA UYGUNLUK VE İHLAL ETMEME GARANTİLERİ DAHİL ANCAK BUNLARLA SINIRLI OLMAYAN AÇIK VEYA ZIMNİ HİÇBİR GARANTİ VERİLMEKSİZİN "OLDUĞU GİBİ" SAĞLANMAKTADIR. HİÇBİR DURUMDA YAZARLAR VEYA TELİF HAKKI SAHİPLERİ, SÖZLEŞME, HAKSIZ VEYA BAŞKA BİR DURUMDA, YAZILIM'IN DIŞINDAKİ VEYA BAĞLANTILI OLARAK KULLANIMINDAN VEYA YAZILIM'LARDAKİ HERHANGİ BİR İDDİA, ZARAR VEYA BAŞKA YÜKÜMLÜLÜKLERDEN SORUMLU DEĞİLDİR.
+</pre></blockquote>
```
[the remark this answers is not in the document]
Comment (bitigchi) [reply CotEditor/tr.lproj/Acknowledgments.html]: Reverted.

**Review by 1024jp — APPROVED** [review]

**Comment by 1024jp** [comment]
Thank you for the update! I've merged your PR.

> 大丈夫、コットエディトールが大好き。いつだってアップデートをします。

Great! Happy to hear that :)


>> Instead, it is kind, if you also make a PR for the App Store description in the Documents-for-AppStore repo.
> 
> します、でも、どうやるか？ Documents on the screenshots are available somewhere to use?

Thank you again. Please add Description.tr.txt and make a PR. Regarding the screenshots, you don't need them. I'll make it when I have time.
